File: app/front/src/features/admin/constants/normative-table-status.ts
import type { NormativeTableStatus } from "@services/normative-tables";

export const normativeTableStatusLabels = {
	DRAFT: "Rascunho",
	PUBLISHED: "Publicada",
	SUPERSEDED: "Substituída",
} as const satisfies Record<NormativeTableStatus, string>;

export const normativeTableStatusBadgeVariants = {
	DRAFT: "warning",
	PUBLISHED: "success",
	SUPERSEDED: "outline",
} as const satisfies Record<NormativeTableStatus, string>;

File: app/back/src/main/java/br/com/ampere/domain/NormativeTableStatus.java
package br.com.ampere.domain;

public enum NormativeTableStatus {
  DRAFT("Aguardando Revisão"),
  PUBLISHED("Vigente"),
  SUPERSEDED("Substituída");

  private final String label;

  NormativeTableStatus(String label) {
    this.label = label;
  }

  public String label() {
    return label;
  }
}
